packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,2207 @@
+---
+title: Devices
+category: management
+---
+
+# Device API
+
+<ApiLifecycle access="ea" />
+
+The Okta Device API provides centralized integration platform to store and manage device information. Okta administrators can use these APIs to manage workfoce identity device object information.
+
+The Device API supports the following **Device Operations**:
+* Create, Get, Update, Delete device objects
+* Perform lifecycle transitions on the device objects
+* Link devices to the devices and manage these associations
```

Comment:
Is this a typo? I'm wondering if it is supposed to say "link users to the devices"